packages/extension-chakra-storefront/src/pages/account/wishlist/partials/wishlist-secondary-button-group.test.js

```diff
@@ -386,3 +389,30 @@ test('can remove item', async () => {
         expect(mockedHandler).toHaveBeenCalled()
     })
 })
+
+test('shows error toast when remove item fails', async () => {
+    // Override the delete handler to return an error
+    prependHandlersToServer([
+        {
+            path: '*/customers/:customerId/product-lists/:listId/items/:itemId',
+            method: 'delete',
+            status: 500,
+            res: () => ({error: 'Server error'})
+        }
+    ])
+
+    const mockedHandler = jest.fn()
+    renderWithProviders(<MockedComponent onClick={mockedHandler} />)
+
+    const removeButton = await screen.findByRole('button', {
+        name: /remove/i
+    })
+    user.click(removeButton)
```

Comment:
user actions is always async. Wrapping it in `act` to avoid React warning about using act()
```suggestion
    await act(() => {await user.click(removeButton)}
```